test/config.json

```diff
@@ -2,7 +2,7 @@
   "tests": [
     {
       "file": "test/specs/chrome.js",
-      "timeout": 20000,
+      "timeout": 38000,
```

Comment:
@SherBhachuOnFido why we need to increase this? Instead, can we selectively increase this timeout for tests that really need to?

Reply:
@ethan605 we need to increase this as some of the displayed error popups only appear after 10/20 seconds. Unless there is a way where, with the mock we can ensure they all appear within a certain time?

There is no really easy way to do this selectively per tests.